### PR TITLE
Fix usage of steam, added logging of json errors

### PR DESCRIPTION
**Pull request by kammerjaeger**
Hi,
Please have a look at the change, currently the wrong stream is used so no errors are shown.
I also added a json error output.

Let me know what you think.
It would also be nice to have a command line option to forward the input to stdout without a script or code around it.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -34,7 +34,10 @@ function start (opts) {
   }
 
   const stream = pinoElasticSearch(opts)
-
+    
+  stream.on('unknown', async (error) => {
```
Comment (mcollina) [inline cli.js:38]:
```suggestion
  stream.on('unknown', (error) => {
```

Why was this async?

**Review by mcollina — COMMENTED** [review]
in what circumstances `unknown` is emitted?

File: cli.js (hunk 1 above)
Comment (kammerjaeger) [reply cli.js:38]: Sorry that was my mistake, to much async programming :-)

**Comment by kammerjaeger** [comment]
According to documentation (and looking into the source code):
"Event received by pino-elasticsearch is unparseable (via JSON.parse)"
I fixed and added the line in the error message.

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by kammerjaeger** [comment]
Looks like the ';' is not liked in the linter, I will remove them.

**Pull request merged**